pennylane/tensorbox/fn.py

```diff
@@ -0,0 +1,416 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Function wrappers for the TensorBox API"""
+# pylint:disable=abstract-class-instantiated
+import warnings
+
+import numpy as np
+
+from .tensorbox import TensorBox
+
+
+def _get_multi_tensorbox(values):
+    """Determines the correct framework to dispatch to given a
+    sequence of tensor like objects.
+
+    Args:
+        values (Sequence[tensor_like]): a sequence of tensor like objects
+
+    Returns:
+        .TensorBox: A TensorBox that will dispatch to the correct framework
+        given the rules of precedence. This TensorBox will contain the *first*
+        tensor like object in ``values`` that corresponds to the highest-priority

```

Comment:
```suggestion
        tensor-like object in ``values`` that corresponds to the highest-priority
```

this statement is a bit confusing?

Reply:
to be more precise: the grammar makes it unclear if values corresponds to the framework, or the object